### PR TITLE
Merge ChanJianHao-Update-AddWs

### LINKED ISSUE #143: Update AddWorkspace command
To update `adduser` to `addws` and make use of the refactored parser.

### PR DESCRIPTION
**Pull request by ChanJianHao**
Adds addws command, resolving #143 

This command makes use of the new parser and should reduce code replication.

### CODE REVIEW

**Review by michaeldinata — APPROVED** [review]
Thanks for updating the parser for addworkspace. LGTM!

File: src/main/java/seedu/duke/command/AddWorkspaceCommand.java [hunk 1]
```diff
@@ -10,38 +10,25 @@
...
-                    if (PARAMETER_NAME.equals(parameterTextSplit[0])) {
-                        name = parameterTextSplit[1];
-                    }
-                }
-            }
-        } catch (IndexOutOfBoundsException e) {
-            throw new AniException(EXCEPTION_INVALID_PARAMETERS);
-        }
+
+    protected String newWorkspaceName;
```
Comment (n3wsoldier) [inline src/main/java/seedu/duke/command/AddWorkspaceCommand.java:14]: I think for command, private will be the better access modifiers, as there will not be any subclass to AddWorkspaceCommand.

Comment (ChanJianHao) [reply src/main/java/seedu/duke/command/AddWorkspaceCommand.java:14]: Updated, thanks!

**Review by n3wsoldier — APPROVED** [review]
LGTM :)

**Pull request merged**